docs/backend/configuration.md

```diff
@@ -40,3 +41,43 @@ object and viewed / modified via `/internal/config` (see
 [Accessing the admin panel](./admin.md)).
 
 ![Screenshot of site configuration admin interface](https://user-images.githubusercontent.com/47985/73627238-6276d500-467e-11ea-8724-afb703f056bc.png)
+
+## Vault
+
+The vault Ruby gem allows us to interact with
+[HashiCorp's Vault](https://www.vaultproject.io/docs/what-is-vault). In a
+nutshell, Vault is a tool for securely storing and accessing secrets. The way we
+have it configured, it is completely optional for running a Forem. To access it
```

Comment:
I think we could just cut `The way we have it configured`, since that doesn't seem particularly relevant to these docs because we don't ever tell anyone how to "un-configure" it :)